core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/MetadataSetupService.java

```diff
@@ -148,6 +148,8 @@ public int saveInstanceMetaData(Stack stack, Iterable<CloudVmMetaDataStatus> clo
                 instanceMetaDataEntry.setInstanceId(instanceId);
                 instanceMetaDataEntry.setPrivateId(privateId);
                 instanceMetaDataEntry.setStartDate(clock.getCurrentTimeMillis());
+                instanceMetaDataEntry.setSubnetId(instanceMetaDataEntry.getSubnetId());
+                instanceMetaDataEntry.setAvailabilityZone(instanceMetaDataEntry.getAvailabilityZone());
```

Comment:
This is not really effective. 😄 Anyway, the respective setter calls were moved into `setupFromCloudInstance()`; see #10944. So these lines can be removed.